api/app/service/rest.py

```diff
@@ -741,6 +746,36 @@ def get_organisation_for_service(service_id):
     return jsonify(organisation.serialize() if organisation else {}), 200
 
 
+@service_blueprint.route('/failing-callback-stats', methods=['GET'])
+def get_callback_failing_stats_for_all_services():
+    stats = dao_get_callback_failing_stats_for_all_services()
+    data = {
+        'totalFailureCount': stats[0],
```

Comment:
you should be able to refer to the label of the column from callback_failures_dao here:

* `stats.total_failure_count` and
* `stats.failed_notification_count`

we use snake_case for API responses

Reply:
Oh cool, updated